views/dashboard-resources.hbs

```diff
@@ -0,0 +1,203 @@
+                <div class="col-md-6">
+                    <p>Ultrasound Proximity Sensor &nbsp
+                    <p>
+                        <a href="https://www.instructables.com/id/Simple-Arduino-and-HC-SR04-Example/">(here)</a>
+                </div>
+            </div>
+            <!-- Motion Add-On -->
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <h3 class="motion-box">MOTION ADD-ON</h3>
+                </div>
+            </div>
+            <!--row 1 -->
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/bzGNTVsfO3E" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!-- Audio & Visual Add-on -->
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <h3 class="AV-box">AUDIO & VISUAL ADD ON</h3>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/UmYiHTOz-5k" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <!-- TI Box -->
+            <div class="row">
+                <div class="col-md-12 text-center">
+                    <h3 class="TI-box">TI BOX</h3>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/QETprOYhmyQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+                <div class="col-md-6 ">
+                    <iframe src="https://www.youtube.com/embed/9hYy_EB1eYw" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+            <div class="row text-center">
+                <div class="col-md-6">
+                    <p>TI Detailed blog &nbsp
+                    <p>
+                        <a href="https://www.element14.com/community/community/design-challenges/texas-instruments-safe-sound-wearables-design-challenge/blog/2016/11/29/wifi-with-msp432-and-simplelink-cc3100">(here)</a>
+                </div>
+                <div class="col-md-6">
+                    <p>Bluetooth Tutorial &nbsp
+                    <p>
+                        <a href="https://gist.github.com/YechengLyu/73e0c0369cebdd4a4859845bfddb89fb">(here)</a>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+</html>
+
+
+
+
+{{!-- 

```

Comment:
Delete this (and the extra newlines) since it doesn't seem to be used

Reply:
good catch